src/backend/src/prisma/schema.prisma

```diff
@@ -640,6 +663,46 @@ model Meeting {
   teamId       String
 }
 
+model Wbs_Proposed_Changes {
+  wbsProposedChangesId       String                         @id @default(uuid())
+  name                       String
+  status                     WBS_Element_Status
+  projectLeadId              Int
```

Comment:
this should be a relation with the projectLead/ manager. Also these two fields should be optional on the wbs proposed changes